### DISCUSSION TITLE
Digital/lab1/verilog_coding/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#6] [Announcements]
# Digital/lab1/verilog_coding/

https://soc.ustc.edu.cn/Digital/lab1/verilog_coding/

**Comment by Wallbreaker5th** [comment]
如果安装 TerosHDL 后尝试打开 Schematic Viewer 没有显示内容，扩展输出 `[error] Yosys failed.` 等信息，可能需要 `sudo pip install teroshdl`。
